.circleci/config.yml

```diff
@@ -545,6 +545,7 @@ jobs:
         - notify_slack_deploy:
             slack_bot_token: $SLACK_BOT_TOKEN
             slack_channel: "acf-ohs-ttahub--contractor-customer-team"
+            target_env: << parameters.target_env >>
```

Comment:
Why did we make changes to config.yml in this pr?

Reply:
Came in when I merged something.